com.github.diamondburned.gtkcord4.yml

```diff
@@ -0,0 +1,40 @@
+id: com.github.diamondburned.gtkcord4
```

Comment:
as per de documentation it's `io.github` you should use.

Reply:
1. the linter will tell you: build will fail
2. https://docs.flatpak.org/en/latest/conventions.html#application-ids

The example you list are grandfather from a time these rules were not enforced.

`diamondburned.github.io` does resolve.

Reply:
fixed with https://github.com/flathub/flathub/pull/3857/commits/92abdb44f1acc2c2985346446dc5b21d328d6957